docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md

```diff
@@ -0,0 +1,63 @@
+<div><h3 class="sub-docs-type" id="bigcommerce-for-wordpress">BigCommerce for WordPress</h3></div>
+
+# BigCommerce for WordPress Cron
+
+This document provides a general description for WP-Cron and how to change its behavior to improve its reliability.
+
+BigCommerce for WordPress plugin has update/sync logic which is based on cron jobs (WP-Cron). WP-Cron is built-in logic for processing postponed and background tasks. While that logic is great and flexible it also has some problems which can affect import product logic.
```

Comment:
Is it accurate to say here that server cron is the BigCommerce for WordPress plugin?

Reply:
It's not a plugin but a server cron job. I'll check with Paul from MT to verify. @bc-andreadao Thanks for all the great feedback!